platform/android/java/build.gradle

```diff
@@ -24,15 +24,15 @@ allprojects {
 ext {
     sconsExt = org.gradle.internal.os.OperatingSystem.current().isWindows() ? ".bat" : ""
 
```

Comment:
I'm not sure if this is a necessary change or not

Reply:
The changes below are not necessary. They are arguments for the `scons` build command hence why the abis are slightly different from what gradle uses.